public/locales/es/translation.json

```diff
@@ -0,0 +1,193 @@
+      "how_to_use": "¿Cómo se usa?",
+      "how_to_use_content_1": "Concéntrate en una relación específica que tengas en tu vida. Luego, echa un vistazo a las palabras escritas en cada panel, y piensa si son relevantes para tu relación, o si te gustaría que lo fueran. Revela los colores de tu relación haciendo clic en los paneles que te gusten.",
+      "how_to_use_content_2": "Exactamente cómo uses esta herramienta depende de ti, pero funciona mejor cuando compartes los resultados con tus vínculos. Puedes usarlo junto con tus vínculos o cada cual usarlo de forma independiente y después compartir los resultados. El punto es tener una conversación.",
+      "whats_ra": "¿Qué es la Anarquía Relacional?",
+      "whats_ra_content": "La Anarquía Relacional es una forma de abordar las relaciones de manera intencional. En lugar de asumir cómo debería ser una relación basándose en su etiqueta (por ejemplo, cónyuge, amante, padre, amistad), defines activamente cómo quieres que sea la relación. Descrita por primera vez por Andie Nordgren en 2006, <a href=\"https://log.andie.se/post/26652940513/the-short-instructional-manifesto-for-relationship\">El Manifiesto Instructivo Corto para la Anarquía Relacional</a> es un buen lugar para comenzar."
+  },
+  "flavours": {
+      "caregiving": "Cuidado",
+      "collaboration": "Colaboración",
+      "communication": "Comunicación",
+      "companionship": "Compañerismo",
+      "creativity": "Creatividad",
+      "domesticity": "Vida doméstica",
+      "each_other": "Le un a le otre",
+      "emotional_intimacy": "Intimidad emocional",
+      "emotional_support": "Apoyo emocional",
+      "exclusivity": "Exclusividad",
+      "finances": "Finanzas",
+      "hierarchy": "Jerarquía",
+      "in_person": "En persona",
+      "kink": "Kink",
+      "labels": "Etiquetas",
+      "legal_arrangements": "Acuerdos legales",
+      "others": "Otres",
+      "our_relationship_includes": "Nuestra relación incluye...",
+      "partnership": "Sociedad",
+      "phone": "Teléfono",
+      "physical_intimacy": "Intimidad física",
+      "power_exchange": "Intercambio de poder",
+      "public_displays_of_affection": "Demostraciones públicas de afecto",
+      "sexuality": "Sexualidad",
+      "societal_constraints": "Limitaciones sociales",
+      "spirituality": "Espiritualidad",
+      "structural_inequality": "Desigualdad estructural",
+      "text": "Texto",
+      "acrobatics": "Acrobacias",
+      "ability": "Capacidad",
+      "class": "Clase",
+      "financial": "Financiero",
+      "gender": "Género",
+      "race": "Raza",
+      "adoption": "Adopción",
+      "animals": "Animales",
+      "being_asked_for_advice": "Ser consultade",
+      "being_vulnerable": "Ser vulnerable",
+      "body_contact": "Contacto corporal",
+      "body_touch": "Toque corporal",
+      "boss_employee": "Jefi - Empleade",
```

Comment:
Is "Jefi" the gender-neutral of "Jefe"? Cool!

Reply:
Yes